charts/cass-operator/values.yaml

```diff
@@ -0,0 +1,66 @@
+# Default values for cass-operator.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# NOTE I have some TODO questions for reviewers, some of which are about
+# auto generated properties.
+
+replicaCount: 1
+
+image:
+  repository: docker.io/datastax/cass-operator
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: master-a45fd80b40e0129e8883a953d595ca85e4b7ce79
+  # TODO It better to keep this commented out or set it to empty string?
+  # registryOverride:
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+
+serviceAccount:
+  # TODO Should we have a create property? Don't we always want to create a service account?
+  # Specifies whether a service account should be created
+  create: true
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+
+# TODO Do we need this for an operator deployment?
```

Comment:
Are there some other tools that might want it? I think at least datadog uses podAnnotations for some health scraping. I don't know why the choice between podAnnotation vs. annotations in general. Also, should that sort of parameter be at top level (add additional annotations to every pod / every container / etc) instead of having to define it for every part separately?

Reply:
Good question. This is part of the auto generated stuff. I think we need to consider what makes the most sense in terms of our operator deployments and not deployments in general.

Reply:
>I don't know why the choice between podAnnotation vs. annotations in general

It makes more sense if you look at the `deployment.yaml` template. The annotations are only added to the pod template, not the deployment itself.